core/src/main/java/com/digitalpebble/stormcrawler/protocol/httpclient/HttpProtocol.java

```diff
@@ -99,24 +107,42 @@ public void configure(final Config conf) {
         String proxyHost = ConfUtils.getString(conf, "http.proxy.host", null);
         int proxyPort = ConfUtils.getInt(conf, "http.proxy.port", 8080);
 
+        String proxyUser = ConfUtils.getString(conf, "http.proxy.user", null);
+        String proxyPass = ConfUtils.getString(conf, "http.proxy.pass", null);
+        String proxyAuthScheme = ConfUtils.getString(conf, "http.proxy.authscheme", AuthSchemes.BASIC);
+        List<String> authSchemes = Lists.newArrayList();
```

Comment:
`= new ArrayList<>();`

takes even less characters and does not require the import of the shaded lib